core/src/components/range/range.tsx

```diff
@@ -142,6 +143,29 @@ export class Range implements ComponentInterface {
    */
   @Prop() ticks = true;
 
+  /**
+   * The start position of the range active bar, only available with a single knob (dualKnobs="false").
+   * Valid values are between the `min` and `max` values.
+   */
+  @Prop() barActiveStart?: number;
+  @Watch('barActiveStart')
```

Comment:
Bikeshedding: We call this the "active bar" in the description. Should we call the prop `activeBarStart` instead?

Reply:
I like it 👍 I'll rework the API.